pages/view/bookmark.tsx

```diff
@@ -1,7 +1,63 @@
+import { css } from '@emotion/react'
+import styled from '@emotion/styled'
 import { NextPage } from 'next'
+import Link from 'next/link'
+import { useMemo } from 'react'
+import BookmarkList from 'components/Bookmark/BookmarkList'
+import { useBookmarks } from 'lib/bookmarkData'
 
 const BookMark: NextPage = () => {
-  return <></>
+  const { data, isLoading } = useBookmarks()
+
+  const bookmarks = useMemo(() => {
+    return data
+  }, [data])
```

Comment:
ここの `useMemo` 意味ないかも
```suggestion
  const { data: bookmarks, isLoading } = useBookmarks()
```